homeassistant/components/nanoleaf/light.py

```diff
@@ -85,6 +76,8 @@ class NanoleafLight(LightEntity):
 
     def __init__(self, light, name, unique_id):
         """Initialize an Nanoleaf light."""
+        self._attr_supported_color_modes = (COLOR_MODE_COLOR_TEMP, COLOR_MODE_HS)
```

Comment:
This should be a set:
```suggestion
        self._attr_supported_color_modes = {COLOR_MODE_COLOR_TEMP, COLOR_MODE_HS}
```